### ISSUE TITLE
SSL error trying to pair

### ISSUE THREAD

**Issue by Aransh** [louis49/homebridge-plugin-androidtv#56]
Trying to connect Xiaomi TV Box S 2nd Gen
UI is acting weird and failing.

Worth mentioning my Xiaomi MiTV 4S is successfully paired.

log shows:

Start Remote Connect

192.168.68.63 Remote secureConnect

192.168.68.63 Remote secureConnect

192.168.68.63 [Error: A09BFDB6:error:0A000416:SSL routines:ssl3_read_bytes:sslv3 alert certificate unknown:../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1605:SSL alert number 46

] {

  library: 'SSL routines',

  reason: 'sslv3 alert certificate unknown',

  code: 'ERR_SSL_SSLV3_ALERT_CERTIFICATE_UNKNOWN'

}

192.168.68.63 Remote Connection closed  false

192.168.68.63 Remote Connection closed  false

**Comment by SawkeeReemo** [comment]
Same issue here. Is this project dead? (trying to pair with NVIDIA Shield Pro)

**Comment by tomeranaray** [comment]
Same issue here. I guess the TV box got updated (remotely by the vendor/cable company), and now refuses connection with the HomeBridge plugin. The problem is it's seems very hard to unpair and repair the TV Box. I can't find any option for it in the plugin.

**Comment by JuanIgnaciosj** [comment]
> Same issue here. I guess the TV box got updated (remotely by the vendor/cable company), and now refuses connection with the HomeBridge plugin. The problem is it's seems very hard to unpair and repair the TV Box. I can't find any option for it in the plugin.

The same situation here, my Box was updated yesterday and I'm not able to pair the device (In hombridge works well) but my app in my iPhone and iPad doesn't find the TV. Anybody could fix this? 

Edit: I got a solution changing the plugin for this https://github.com/dwaan/homebridge-adb 
If this plugin works for you, the new has the same use and is really easy to install (you will have the same functions and you can add others)

**Comment by SawkeeReemo** [comment]
How did you get that to work? I’m running Homebridge in a docker container, and nothing I’ve tried works. It’s literally the only plugin I’ve never figured out how to make work. Followed all of the directions, network is good, but it just doesn’t connect to my Shield.

**Comment by JuanIgnaciosj** [comment]
Like I said, I changed the plugin for that I posted before. Follow the instructions, and It will work only is a bir slow than this.
In my case Btw I had static IP assigned for my devices, look at that, maybe that solves yoyr problem.

**Comment by Aransh** [comment]
@JuanIgnaciosj That's the plugin I'm currently using, if you look at the bottom of the link you've sent, you'll see the ads developer is recommending this plugin instead (which I hoped would resolve some of the issues I'm having with the adb plugin, but since it doesn't work at all...)

**Comment by SawkeeReemo** [comment]
> Like I said, I changed the plugin for that I posted before. Follow the instructions, and It will work only is a bir slow than this. In my case Btw I had static IP assigned for my devices, look at that, maybe that solves yoyr problem.

Yes, I know. Many of us cannot get the homebridge-adb plugin to work. Yes, I’ve followed all the instructions. Some folks say that for whatever reason, you can’t get the latest version of Android Tools in the homebridge docker container. So I was asking how you got it to work. 

EDIT: for the NVIDIA Shield Pro.

**Comment by JuanIgnaciosj** [comment]
You tried to install from the shell of the docker? (Not homebridge?) and after that mapping the installation place?

**Comment by SawkeeReemo** [comment]
I can get into the container’s shell of course, follow all the instructions on their guide, no problem. No idea how what “map the installation place” means though. Following their guide doesn’t work, unfortunately. It’s either outdated or they are missing something. Lots of people having this issue.
